README.md

```diff
@@ -106,6 +106,44 @@ If you need to use basic authentication to connect with the Elasticsearch cluste
 ```
 cat log | pino-elasticsearch --node https://user:pwd@localhost:9200
 ```
+
+You can include the `auth` field in your configuration like so:
+```js
+const pinoElastic = require('pino-elasticsearch')
+
+const streamToElastic = pinoElastic({
+  index: 'an-index',
+  consistency: 'one',
+  node: 'http://localhost:9200',
+  auth: {
+    username: 'user',
+    password: 'pwd'
+  },
+  'es-version': 7,
+  'flush-bytes': 1000
+})
+```
+
+And if you're using elastic cloud, you can pass an `apiKey` instead:
```

Comment:
This is not entirely correct, you can use api key also without elastic cloud and you can use basic auto with elastic cloud as well :)

Reply:
Ah, shows my lack of comprehensive knowledge of elastic! Thank you for pointing this out.